config/routes.rb

```diff
@@ -3,9 +3,13 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
-  # https://github.com/heartcombo/devise/blob/main/README.md
+    # https://github.com/heartcombo/devise/blob/main/README.md
     devise_for :users
 
+    # Profile routes
+    get "/profiles/me", to: "profiles#show_me"
+    get "/profiles/:id", to: "profiles#show", as: "profile"
```

Comment:
Since `"profile"` is an identifier and not a mutable or complex string, it would be preferable to use `:profile`, i.e. Ruby's symbol type instead.

Reply:
I see the Ruby guidelines aren't consistent on this: https://guides.rubyonrails.org/routing.html#overriding-the-named-route-helpers, https://guides.rubyonrails.org/routing.html#naming-routes.

Could you elaborate on when to use one over the other, and what the tradeoffs are?

Reply:
Hm... unsure about the tradeoffs, but I just looked at a list of open source projects using Ruby (went on https://github.com/asyraffff/Open-Source-Ruby-and-Rails-Apps and clicked on names that sounded familiar to me) and they all use symbols, which indicates for me that they are more Ruby-idiomatic:

- https://github.com/mastodon/mastodon/blob/main/config/routes.rb#L50
- https://github.com/forem/forem/blob/main/config/routes.rb#L15
- https://github.com/gitlabhq/gitlabhq/blob/master/config/routes.rb#L86
- https://github.com/discourse/discourse/blob/main/config/routes.rb#L160
- https://github.com/openstreetmap/openstreetmap-website/blob/master/config/routes.rb#L19

Reply:
Sounds good 👍
```suggestion
    get "/profiles/:id", to: "profiles#show", as: :profile
```